web-ui/src/main/resources/catalog/components/search/searchfiltertag/SearchFilterTagsDirective.js

```diff
@@ -3,463 +3,206 @@
 
   goog.require('gn_thesaurus_service');
 
-  var module = angular.module('search_filter_tags_directive',
-      ['gn_thesaurus_service', 'pascalprecht.translate']);
-  module.filter('decodeURIComponent', function() {
```

Comment:
It looks like this breaks stuff in editor file store
![image](https://user-images.githubusercontent.com/1701393/74024961-31aade80-49a3-11ea-998b-5e6fe159b05a.png)